packages/bezier-react/src/components/Forms/RadioGroup/RadioGroup.tsx

```diff
@@ -28,7 +28,7 @@ function RadioGroupImpl<Value extends string>({
         direction={direction}
       >
         { React.Children.map(children, (child, index) => (
-          <StackItem key={child?.key ?? index}>
+          <StackItem key={(child as React.ReactElement)?.key ?? index}>
```

Comment:
```suggestion
          <StackItem key={isValidElement(child) ? child.key : index}>
```
이 어떨까요? `formGroup` 도 동일한 assertion 이 있어서 바꾼다면 같이 바꾸는 게 좋을 것 같네요!